adam-core/src/main/scala/org/bdgenomics/adam/models/ReferenceRegion.scala

```diff
@@ -85,6 +102,23 @@ case class ReferenceRegionWithOrientation(region: ReferenceRegion,
   def end: Long = region.end
 }
 
+object ReferenceRegionContext {
+  implicit val referenceRegionOrdering = ReferenceRegionOrdering
+  implicit val referenceRegionWithOrientationOrdering = ReferenceRegionWithOrientationOrdering
```

Comment:
I _think_ but cannot prove that it would fail to build. @tdanford is probably the best person to comment, but I do think this is necessary for #511, and I do agree that #511 is something that should be fixed. We'd need to fix #511 if we were to move `ReferenceRegion` into Avro, for instance.